src/Init/Data/Array/Subarray.lean

```diff
@@ -44,10 +44,10 @@ def get (s : Subarray α) (i : Fin s.size) : α :=
    exact Nat.add_lt_of_lt_sub this
   s.array[s.start + i.val]
 
-instance : GetElem (Subarray α) Nat α fun xs i => i < xs.size where
+instance : GetElemBase (Subarray α) Nat α fun xs i => i < xs.size where
   getElem xs i h := xs.get ⟨i, h⟩
 
-instance : LawfulGetElem (Subarray α) Nat α fun xs i => i < xs.size where
+instance : GetElem (Subarray α) Nat α fun xs i => i < xs.size where
```

Comment:
This instance is no longer necessary with `getElemOfDecidable`.

Reply:
This is the base one. It's `GetElem?` that's not needed anymore!